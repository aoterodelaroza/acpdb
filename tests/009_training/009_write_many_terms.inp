## check: 009_write_many_terms.out -a1e-10
## delete: 009_write_many_terms.db
## delete: 009_write_many_terms-set.h2o_h2o_1@h_l_1.gjf
## delete: 009_write_many_terms-set.h2o_h2o_1@h_l_2.gjf
## delete: 009_write_many_terms-set.h2o_h2o_2@h_l_1.gjf
## delete: 009_write_many_terms-set.h2o_h2o_2@h_l_2.gjf
## delete: 009_write_many_terms-set.h2o_h2o@h_l_1.gjf
## delete: 009_write_many_terms-set.h2o_h2o@h_l_2.gjf
## delete: 009_write_many_terms-set.nh3_nh3_1@h_l_1.gjf
## delete: 009_write_many_terms-set.nh3_nh3_1@h_l_2.gjf
## delete: 009_write_many_terms-set.nh3_nh3_2@h_l_1.gjf
## delete: 009_write_many_terms-set.nh3_nh3_2@h_l_2.gjf
## delete: 009_write_many_terms-set.nh3_nh3@h_l_1.gjf
## delete: 009_write_many_terms-set.nh3_nh3@h_l_2.gjf
## labels: regression quick

system rm -f 009_write_many_terms.db
connect 009_write_many_terms.db

insert method m_empty
end
insert method m_ref
end
insert set 009_write_many_terms-set
 din ../zz_source/s22partial.din
 directory ../zz_source/s22_xyz
 method m_ref
end
insert evaluation
 method m_empty
 property 009_write_many_terms-set.nh3_nh3
 value 0.2
end
insert evaluation
 method m_empty
 property 009_write_many_terms-set.h2o_h2o
 value 0.3
end
insert term
 method m_empty
 property 009_write_many_terms-set.h2o_h2o
 atom H
 L 0
 exponent 0.1
 value 0.3
end
insert term
 method m_empty
 property 009_write_many_terms-set.h2o_h2o
 atom H
 L 0
 exponent 0.2
 value 0.3
end
insert term
 method m_empty
 property 009_write_many_terms-set.nh3_nh3
 atom H
 L 0
 exponent 0.1
 value 0.3
end
insert term
 method m_empty
 property 009_write_many_terms-set.nh3_nh3
 atom H
 L 0
 exponent 0.2
 value 0.3
end

training
 atom H l
 exp 0.1 0.2
 empty m_empty
 reference m_ref
 subset
  set 009_write_many_terms-set
 end
end

write
 template ../zz_source/templates/gau2.gjf
 training
 term
end
