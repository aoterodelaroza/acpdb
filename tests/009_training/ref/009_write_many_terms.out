%% verbose
%% system rm -f 009_write_many_terms.db
* SYSTEM: rm -f 009_write_many_terms.db

%% connect 009_write_many_terms.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 009_write_many_terms.db
Creating skeleton database 

%% insert method m_empty
* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_empty

%% insert method m_ref
* INSERT: insert data into the database (METHOD)
# INSERT METHOD m_ref

%% insert set 009_write_many_terms-set
* INSERT: insert data into the database (SET)
# INSERT SET 009_write_many_terms-set
# INSERT STRUCTURE 009_write_many_terms-set.nh3_nh3
# INSERT STRUCTURE 009_write_many_terms-set.nh3_nh3_1
# INSERT STRUCTURE 009_write_many_terms-set.nh3_nh3_2
# INSERT PROPERTY 009_write_many_terms-set.nh3_nh3
# INSERT EVALUATION (method=m_ref;property=009_write_many_terms-set.nh3_nh3)
# INSERT STRUCTURE 009_write_many_terms-set.h2o_h2o
# INSERT STRUCTURE 009_write_many_terms-set.h2o_h2o_1
# INSERT STRUCTURE 009_write_many_terms-set.h2o_h2o_2
# INSERT PROPERTY 009_write_many_terms-set.h2o_h2o
# INSERT EVALUATION (method=m_ref;property=009_write_many_terms-set.h2o_h2o)

%% insert evaluation
* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=009_write_many_terms-set.nh3_nh3)

%% insert evaluation
* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=m_empty;property=009_write_many_terms-set.h2o_h2o)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=009_write_many_terms-set.h2o_h2o;atom=H;l=0;exponent=0.1)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=009_write_many_terms-set.h2o_h2o;atom=H;l=0;exponent=0.2)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=009_write_many_terms-set.nh3_nh3;atom=H;l=0;exponent=0.1)

%% insert term
* INSERT: insert data into the database (TERM)
# INSERT TERM (method=m_empty;property=009_write_many_terms-set.nh3_nh3;atom=H;l=0;exponent=0.2)

%% training
* TRAINING: started defining the training set 

%% atom H l
%% exp 0.1 0.2
%% empty m_empty
%% reference m_ref
%% subset
%% end
* TRAINING: fininshed defining the training set 

## Description of the training set
# List of atoms and maximum angular momentum channels (1)
| Atom | lmax |
| H | l |

# List of exponents (2)
| id | exp |
| 0 | 0.1 |
| 1 | 0.2 |

# List of subsets (1)
| id | alias | db-name | db-id | initial | final | size | dofit? | litref | description |
| 0 | 009_write_many_terms-set | 009_write_many_terms-set | 1 | 1 | 2 | 2 | 1 |  |  |

# List of methods
| type | name | id | for fit? |
| reference | m_ref | 2 | n/a |
| empty | m_empty | 1 | n/a |

# List of properties (2)
| fit? | id | property | propid | alias | db-set | proptype | nstruct | weight | refvalue |
| yes | 1 | 009_write_many_terms-set.nh3_nh3 | 1 | 009_write_many_terms-set | 009_write_many_terms-set | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 2 | 009_write_many_terms-set.h2o_h2o | 2 | 009_write_many_terms-set | 009_write_many_terms-set | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |

# Calculation completion for the current training set
# Reference: 2/2 (complete)
# Empty: 2/2 (complete)
# Terms: 
| H | l | 0.1 | 2/2 | (complete)
| H | l | 0.2 | 2/2 | (complete)
# Total terms: 4/4 (complete)

%% write
* WRITE: write input files for database structures

# WRITE file ./009_write_many_terms-set.h2o_h2o_2@h_l_1.gjf
# WRITE file ./009_write_many_terms-set.h2o_h2o_1@h_l_1.gjf
# WRITE file ./009_write_many_terms-set.h2o_h2o@h_l_1.gjf
# WRITE file ./009_write_many_terms-set.nh3_nh3_2@h_l_1.gjf
# WRITE file ./009_write_many_terms-set.nh3_nh3_1@h_l_1.gjf
# WRITE file ./009_write_many_terms-set.nh3_nh3@h_l_1.gjf
# WRITE file ./009_write_many_terms-set.h2o_h2o_2@h_l_2.gjf
# WRITE file ./009_write_many_terms-set.h2o_h2o_1@h_l_2.gjf
# WRITE file ./009_write_many_terms-set.h2o_h2o@h_l_2.gjf
# WRITE file ./009_write_many_terms-set.nh3_nh3_2@h_l_2.gjf
# WRITE file ./009_write_many_terms-set.nh3_nh3_1@h_l_2.gjf
# WRITE file ./009_write_many_terms-set.nh3_nh3@h_l_2.gjf

